src/shared/conf-parser.c

```diff
@@ -1461,6 +1461,45 @@ int config_parse_permille(
         return 0;
 }
 
+int config_parse_io_cost_qos_scaling_percentage(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        loadavg_t *io_cost_qos_scaling_percentage = data;
+        loadavg_t result;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(io_cost_qos_scaling_percentage);
+
+        r = parse_loadavg_fixed_point(rvalue, &result);
```

Comment:
this is not a load average, is it?

Use parse_percent(), parse_permille() or parse_permyriad()